src/assets/scss/_input-switch.scss

```diff
@@ -171,16 +171,114 @@
   }
 }
 
-.input-switch-boolean {
+// rule for foreignkey dropdown with search input to remove space at the top of the dropdown menu above the search input
+.input-switch-foreignkey.fk-dropdown .responsive-dropdown-menu.dropdown-menu {
+  padding-top: 0px;
+}
+
+.input-switch-boolean,
+.input-switch-foreignkey.fk-dropdown {
 
   // remove the default bootstrap caret
-  .dropdown-toggle::after {
+  .dropdown-toggle.no-caret::after {
     content: none !important;
   }
 
-  // make sure the menu expands the whole row
-  .dropdown-menu {
-    width: 100%;
+  // highlight the recordedit input for visual clarity
+  .dropdown.show {
+
+    >.chaise-input-group,
+    .dropdown-menu {
+      border: 2px solid map-get(variables.$color-map, 'primary');
+      border-radius: 6px;
+    }
+  }
+
+  .responsive-dropdown-menu, .dropdown-menu {
+    // make sure the menu expands the whole row
+    width: auto;
+    min-width: 100%;
+    min-height: 75px;
+    background-color: #eef6ff;
+
+    // dropdown opens down
+    &[data-popper-placement^="bottom"] {
+      margin-top: -4px;
+    }
+
+    // dropdown opens up
+    &[data-popper-placement^="top"] {
+      margin-bottom: -3px;
+    }
+
+    // remove rounded borders for search input components when in a dropdown
+    .search-row .chaise-search-box {
+      .chaise-input-group-prepend > .chaise-input-group-text {
+        border-bottom-left-radius: 0;
+      } 
+
+      .chaise-input-group-append > .chaise-search-btn {
+        border-bottom-right-radius: 0;
+      } 
+    }
+
+    // for making the dropdown list scrollable (keeping the buttons sticky)
+    .dropdown-list {
+      // rules specifically for fk-dropdown list of options
+      max-height: 355px;
```

Comment:
This was to make sure the dropdown doesn't get "too long". I want the overall height of the dropdown to be a maximum of `400 px`. This is specifically for the list of options that doesn't include the search box and any padding added above/below the `dropdown-list`.

Otherwise, there is no specific reason.